docs/components/StepperView.jsx

```diff
@@ -113,13 +126,14 @@ export default class StepperView extends PureComponent {
             <Grid>
               <Row grow>
                 <Col span={4}>
-                <Stepper
-                  className="ExampleStepper"
-                  currentStepID={currentStepID}
-                  steps={steps}
-                  sticky={sticky}
-                  onStepClick={seekable ? (id) => this.jumpToStep(id) : null }
-                />
+                <StickyContainer className={classnames(cssClass.STICKY_CONTAINER)}>
```

Comment:
we have Dewey set up so that whatever appears within the `<ExampleCode>` tags gets output verbatim as the code sample that people will refer to.

the way this is currently written, we're going to end up with only the `<Grid>` and `<StickyContainer>` stuff showing, but not the stepper itself.

let's clean this up a bit:
- no need to include the sticky container - doesn't add value to the demo code at this point
- move the `{this._renderConfig()}` part outside of the `<ExampleCode>` element
- [opt] do away with `<Grid>` and `<Row>` - the Grid component can be very noisy and, in most cases, we can accomplish the same thing with regular divs. in this case, would also be fine just to use the `<Col>` elements on their own and drop the excess Grid/Row code